build-system/release-tagger/create-release.js

```diff
@@ -186,22 +193,33 @@ function _createBody(head, base, prs) {
      <h2>Changes by component</h2>
      ${components.join('')}\
    `;
-  return template;
+
+  const cherrypickHeader = head.endsWith('0')
+    ? ''
+    : dedent`\
+    <h2>🌸 Cherry-picked release 🌸</h2>
+    <a href="https://github.com/ampproject/amphtml/releases/tag/${base}">\
+    ${base}</a> was patched and published as <b>${head}</b>. Refer to the \
+    <a href="https://amp-release-calendar.appspot.com">release calendar</a> \
+    for additional channel information.\n\n`;
+  return cherrypickHeader + template;
 }
 
 /**
  * Main function
- * @param {string} tag
- * @param {string} previousTag
+ * @param {string} head
+ * @param {string} base
+ * @param {string} channel
  * @return {Promise<Object>}
  */
-async function main(tag, previousTag) {
-  const {'target_commitish': commit} = await getRelease(tag);
-  const {'target_commitish': previousCommit} = await getRelease(previousTag);
-  const prs = await getPullRequestsBetweenCommits(commit, previousCommit);
-  const body = _createBody(tag, previousTag, prs);
-  return await createRelease(tag, commit, body);
+async function main(head, base, channel) {
+  const {'target_commitish': headCommit} = await getRelease(head);
+  const {'target_commitish': baseCommit} = await getRelease(base);
+  const prs = await getPullRequestsBetweenCommits(headCommit, baseCommit);
```

Comment:
Maybe parallelize these similar to https://github.com/ampproject/amphtml/pull/35692#discussion_r690532164?

Reply:
Hmm, here's how it would look. A little verbose, I'd rather keep it as is.

```suggestion
  const [headRelease, baseRelease] = await Promise.all([
    await getRelease(head),
    await getRelease(base),
  ]);
  const headCommit = headRelease['target_commitish'];
  const baseCommit = baseRelease['target_commitish'];
  const prs = await getPullRequestsBetweenCommits(headCommit, baseCommit);
```